README.rst

```diff
@@ -110,6 +110,14 @@ calling:
 
   cmake --list-presets
 
+Navigate back to the ``libqi-python`` directory
```

Comment:
You're completely right. The instructions below are about building libqi-python native code for development or testing purposes, and not building the wheel. So there should be a section on the same level as the "Wheel" once below.

It's not that much about building "C++ bindings", it's just a different process and purpose.